component/src/main/java/me/androidbox/component/login/LoginScreen.kt

```diff
@@ -0,0 +1,138 @@
+package me.androidbox.component.login
+
+import android.util.Log
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.foundation.text.ClickableText
+import androidx.compose.material3.Text
+import androidx.compose.runtime.*
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.*
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import me.androidbox.component.R
+import me.androidbox.component.general.PasswordEntry
+import me.androidbox.component.general.TaskButton
+import me.androidbox.component.general.UserInputEntry
+import me.androidbox.component.ui.theme.BusbyTaskyTheme
+
+@Composable
+fun LoginScreen(
+    modifier: Modifier = Modifier
+) {
+    var username by remember {
+        mutableStateOf("")
+    }
+
+    val isPasswordVisible = remember {
+        mutableStateOf(false)
+    }
+
+    var password by remember {
+        mutableStateOf("")
+    }
```

Comment:
For all states in your app, I'd recommend using a ViewModel as stated in a previous comment. All these states won't survive configuration changes like screen rotations and will be reset after that. Using a ViewModel avoids that and gives you a central place to easily manage and test your state mapping logic